packages/plugin-core/src/components/pods/PodControls.ts

```diff
@@ -301,6 +314,63 @@ export class PodUIControls {
     return id;
   }
 
+  /**
+   * Prompts a lookup that will determine the scope of export.
```

Comment:
nit: amend description to 

"Prompts a lookup control that allows user to select notes for export."

When I initially read this, I thought this allowed users to select the export scope (as in the enum options of clipboard vs Note vs Hierarchy vs Vault)